public/docs/ts/latest/guide/testing.jade

```diff
@@ -2295,7 +2387,7 @@ table(width="100%")
         [SystemJS](https://github.com/systemjs/systemjs/blob/master/README.md) 
         loads the application and test files.
         This script tells SystemJS where to find those files and how to load them.
-        It's the same version of `systemjs.config.js` used by Setup-based applications.
+        It's the same version of `systemjs.config.js` used by setup-based applications.
```

Comment:
It's the same version of `systemjs.config.js` you installed during [setup](setup.html).